carbon-design-system/carbon-website
# fix: form usage tab image typo

Closes #2676 

**Changed**
- Fixed a typo in the required form image example on the Form usage tab.

----

**Testing**
- Go to `Components` --> `Form` usage tab --> `Optional vs. Required` section
  - Make sure the typo in the image is fixed.
